lib/runner.js

```diff
@@ -183,6 +184,25 @@ Runner.prototype.checkGlobals = function(test){
 };
 
 /**
+ * Randomize tests in a suite.
+ *
+ * @param seed
+ * @return {Runner} for chaining
+ * @api public
+ */
+
+Runner.prototype.randomize = function(mode, seed){
+  debug('randomize mode: "' + mode + '"; seed: ' + seed);
+  if (mode !== 'tests') return this;
+  this._randomize = function(suite){
```

Comment:
this is kind of odd... please just set some flag in `Mocha.prototype.run` on the runner; then check for the flag below in `Runner.prototype.runTests`. there's a precedent for that, anyway